### ISSUE TITLE
Does lispy have anything like paredit-backward-down (C-M-p)?

### ISSUE THREAD

**Issue by bpstahlman** [abo-abo/lispy#300]
I looked through the documentation and didn't see a way to move backwards and downwards into a list: something like paredit-backward-down (C-M-p). Actually, it appears to be missing paredit-forward-down (C-M-d) as well. Did I overlook these commands, or do other lispy commands somehow obviate the need for them?

**Comment by abo-abo** [comment]
These two were left out because they bring you out of special. You can simulate them with <kbd>f</kbd>, <kbd>3 SPC</kbd> and <kbd>2 SPC</kbd>.

**Comment by bpstahlman** [comment]
Not sure what you mean by "they bring you out of special". I mean, I know
what "special" is, but why would they *have* to bring you out of special?

How would 'special-lispy-tab' help?

IIUC, `2 SPC' just puts you in insert mode inside the preceding form, but
wouldn't give you an easy way to move several levels backwards and down. Is
this correct, or did I miss something?

Thanks,
Brett S.

**Comment by abo-abo** [comment]
Let me rephrase:

## paredit-backward-down

Is a combination of:

1. <kbd>f</kbd> from the "after-paren" position to flow inside a list.
2. <kbd>C-f</kbd> (I think <kbd>2 SPC</kbd> is better to step out of special in this case).

## paredit-forward-down

Is a combination of:

1. <kbd>f</kbd> from the "before-paren" position to flow inside a list.
2. <kbd>C-b</kbd> (I think <kbd>2 SPC</kbd> is better to step out of special in this case).

For me, it seems advantageous to use special-only commands (commands that begin and end in special state) as much as possible.  Most important reasons:

1. You can press <kbd>b</kbd> to cancel a navigation command.
2. You can press <kbd>u</kbd> to cancel a modification command.

`paredit-backward-down` and `paredit-forward-down` don't seem to do more than <kbd>f</kbd> to be worth it.
